libs/libcommon/src/libcommon/utils.py

```diff
@@ -113,7 +114,13 @@ def orjson_default(obj: Any) -> Any:
         # the bytes are encoded with base64, and then decoded as utf-8
         # (ascii only, by the way) to get a string
         return base64.b64encode(obj).decode("utf-8")
-    raise TypeError
+    if isinstance(obj, np.integer):
+        return int(obj)
+    if isinstance(obj, np.floating):
+        return float(obj)
+    if isinstance(obj, np.ndarray):
+        return obj.tolist()
```

Comment:
To support the serialization of `numpy` types, why don't you use the `orjson.OPT_SERIALIZE_NUMPY` option in `orjson.dumps`?

Reply:
Thanks for the suggestion! I will send a new commit with this option.